app/videos/[playbackId]/Comments.js

```diff
@@ -0,0 +1,36 @@
+import { Suspense } from 'react'
+
+async function CommentList({ asset }) {
+  const response = await fetch(
+    `https://reactathon-workshop-functions.vercel.app/api/comments/${asset}`,
+  )
+  const data = await response.json()
+
+  return (
+    <ul>
+      {data.comments.map((comment) => {
+        return (
+          <li key={comment.id}>
+            <p>{comment.content}</p>
+            <p>
+              <i>{comment.updated_at}</i>
```

Comment:
Let's display a date here instead of a timestamp.

```suggestion
              <i>{(new Date(comment.updated_at)).toLocaleDateString()}</i>
```